src/psykal/psy.f90

```diff
@@ -2,48 +2,78 @@ module psy_mod
 !
 implicit none
 !
+private
+public :: set_bounds
+public :: zero_top_layer, zero_bottom_layer, multiply_bottom_layer
+public :: zind_psy, zwxy_psy, zslpxy_psy, zslpxy_update_psy, zwxy2_psy, &
+          mydomain_update_psy, zwx_psy, zslpx_psy, zslpx_update_psy, &
+          zwx2_psy, mydomain_psy
+!
+integer :: jpk, jpj, jpi
+!
 contains
   !
-  subroutine zero_layer(field,jpj,jpi)
+  subroutine set_bounds(jpk_in,jpj_in,jpi_in)
```

Comment:
Slightly counter-intuitive argument ordering here but will let it go since this routine is a temporary work-around prior to us getting the infrastructure up and running.